nemo/collections/asr/models/msdd_models.py

```diff
@@ -670,9 +668,9 @@ def __init__(self, cfg_diar_infer: DictConfig, cfg_msdd_model: DictConfig):
         self.clus_diar_model = None
         self.msdd_model = None
         self.spk_emb_state_dict = {}
-
-        self.run_clus_from_loaded_emb = False
-        self.use_speaker_model_from_ckpt = True
+        self.use_speaker_model_from_ckpt = self.cfg_diar_infer.diarizer.msdd_model.parameters.get(
```

Comment:
what is this fix for?

Reply:
There was a hydra config key that I needed to change when I was merging everything into one